src/util.py

```diff
@@ -4,16 +4,20 @@
 
 import gzip
 import hashlib
+from _hashlib import HASH as _Hash
```

Comment:
What about leaving functions that use this type untyped? This looks like an implementation detail on which I think we shouldn't depend.

Reply:
Yes, you're probably right.